snuba/datasets/schema.py

```diff
@@ -94,15 +137,25 @@ def get_local_table_definition(self) -> str:
             self._get_local_engine()
         )
 
-    def _get_local_engine(self):
+    def _get_local_engine(self) -> str:
```

Comment:
I didn't really think about this before, but shouldn't this class also technically be an ABC, that requires this method to be defined?

Reply:
it should. Will change.